### ISSUE TITLE
Incorrect formula for mode in Beta Distribution

### ISSUE THREAD

**Issue by omar-aguilar** [jstat/jstat#133]
Hi :)

Looks like for Beta distribution, it's using the same formula for mode and variance, but that should not be the case

```javascript
  mode: function mode(alpha, beta) {
    return (alpha * beta) / (Math.pow(alpha + beta, 2) * (alpha + beta + 1));
  },
  ...
  variance: function variance(alpha, beta) {
    return (alpha * beta) / (Math.pow(alpha + beta, 2) * (alpha + beta + 1));
  }
```

Regards

**Comment by trevnorris** [comment]
Hm. That does appear to be a problem. Have a solution?

**Comment by jamescgibson** [comment]
Patched in #134! Many thanks for the bug report, @omar-aguilar.

**Comment by trevnorris** [comment]
Fixed by 7e5d94f. Working on release, but npm is having some issues.
